From https://ftp.gnu.org/gnu/bash/bash-5.0-patches/bash50-009

Signed-off-by: Pascal de Bruijn <p.debruijn@unilogic.nl>

			     BASH PATCH REPORT
			     =================

Bash-Release:	5.0
Patch-ID:	bash50-009

Bug-Reported-by:	chet.ramey@case.edu
Bug-Reference-ID:
Bug-Reference-URL:

Bug-Description:

The history file reading code doesn't close the file descriptor open to
the history file when it encounters a zero-length file.

Patch (apply with `patch -p0'):

*** ../bash-5.0-patched/lib/readline/histfile.c	2018-06-11 09:14:52.000000000 -0400
--- b/lib/readline/histfile.c	2019-05-16 15:55:57.000000000 -0400
***************
*** 306,309 ****
--- 312,316 ----
      {
        free (input);
+       close (file);
        return 0;	/* don't waste time if we don't have to */
      }
*** ../bash-5.0/patchlevel.h	2016-06-22 14:51:03.000000000 -0400
--- b/patchlevel.h	2016-10-01 11:01:28.000000000 -0400
***************
*** 26,30 ****
     looks for to find the patch level (for the sccs version string). */
  
! #define PATCHLEVEL 8
  
  #endif /* _PATCHLEVEL_H_ */
--- 26,30 ----
     looks for to find the patch level (for the sccs version string). */
  
! #define PATCHLEVEL 9
  
  #endif /* _PATCHLEVEL_H_ */
